docs/_index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
```

Comment:
We don't generate the HTML documentation, only the reference. There is `rake strings:generate:reference` just for that.

Reply:
removed: https://github.com/voxpupuli/puppet-sftp_jail/pull/70/commits/7df254831960a96013ab8a32c9ef92c9a81c9824